untranslated-messages.txt

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
Is this file required.

Reply:
No. Its not required. I just add that to check whether there is untranslated labels or not. I'll remove that.